### DISCUSSION TITLE
Error during brew installation

### DISCUSSION THREAD

**Discussion by komorniq1985** [Homebrew/discussions#2398] [Getting started]
Hello.
I have the following problem with the installation of brew.
After pasting this line in the terminal

$: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

I get the information

**Illegal variable name**.

And I can't do anything.

After typing n the terminal
$: brew
$: brew: command not found.

Is there any other way to install brew ?
Where should I put the files in case.

I would like to add that I have MacOS Monterey installed

Greetings

**Comment by carlocab** [comment]
Did you follow the `Next Steps` shown to you by the installer? It should've looked something like
```
- Add Homebrew to your PATH in ~/.zprofile:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> ~/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
```

**Reply by komorniq1985** [reply]
@carlocab I already dealt with it.
Thank you for your help.
Greetings

**Reply by silabs-JasonH** [reply]
@komorniq1985 May I know how did you fix the issue?

**Reply by bodonibdoy** [reply]
I have the same issue. How do you fix it?

**Reply by SMillerDev** [reply]
Did you do what this answer suggests?

**Reply by hngdog** [reply]
Same error right at the first install command. Now what. Can't be any 'next steps' if the first step fails!

**Reply by hngdog** [reply]
OK. launch the bash shell first:
`/bin/bash`

then run the install command:
`bash -c $(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)`
